static/js/components/Group.jsx

```diff
@@ -47,61 +200,378 @@ const Group = () => {
+            >
+              <Typography className={classes.heading}>
+                Alert streams and filters
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails className={classes.accordion_details}>
+              <List component="nav" className={classes.padding_bottom}>
+                {group.streams.map((stream) => (
+                  <div key={stream.name}>
+                    <ListItem>
+                      <ListItemText primary={stream.name} />
+                    </ListItem>
+                    <List component="nav" disablePadding>
+                      {group.filters.map(
+                        (filter) =>
+                          filter.stream_id === stream.id ? (
+                            <ListItem
+                              button
+                              component="a"
+                              key={filter.id}
+                              href="#"
+                            >
+                              <ListItemText
+                                className={classes.nested}
+                                primary={filter.name}
+                              />
+                              {(currentUser.roles.includes("Super admin") ||
+                                currentUser.roles.includes("Group admin")) && (
+                                <ListItemSecondaryAction>
+                                  <IconButton
+                                    edge="end"
+                                    aria-label="delete"
+                                    onClick={async () => {
+                                      const result = await dispatch(
+                                        filterActions.deleteGroupFilter({
+                                          filter_id: filter.id,
+                                        })
+                                      );
+                                      if (result.status === "success") {
+                                        dispatch(
+                                          showNotification(
+                                            "Deleted filter from group"
+                                          )
+                                        );
+                                      }
+                                      dispatch(
+                                        groupActions.fetchGroup(loadedId)
+                                      );
+                                    }}
+                                  >
+                                    <DeleteIcon />
+                                  </IconButton>
+                                </ListItemSecondaryAction>
+                              )}
+                            </ListItem>
+                          ) : (
+                            ""
+                          ) /* <div style={{paddingLeft: "2em"}}>No filters defined on stream</div> */
```

Comment:
Should this commented-out line still be here?

Reply:
Removed